<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTransferWidget</class>
 <widget class="QWidget" name="NewTransferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>246</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KMessageWidget" name="errorWidget" native="true">
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="urlLabel">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KUrlComboRequester" name="destRequester"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="groupLabel">
       <property name="text">
        <string>Transfer group:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KComboBox" name="groupComboBox"/>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="KLineEdit" name="urlRequester">
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>247</width>
         <height>42</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QWidget</extends>
   <header location="global">KMessageWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
